<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:#C7A17E;</string>
  </property>
  <widget class="QLabel" name="id_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #dadada;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="so_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Степень обжарки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Название:</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Цена:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3d">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;</string>
   </property>
   <property name="text">
    <string>Впишите данные ниже</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2d">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(213, 233, 255);
border-radius:11px;}

QPushButton:hover{background-color: rgb(213, 233, 200);color: #ffffff}
</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtond">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(213, 233, 255);
border-radius:11px;}

QPushButton:hover{background-color: rgb(213, 233, 200);color: #ffffff}</string>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="z_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Молотый/в зернах:</string>
   </property>
  </widget>
  <widget class="QLabel" name="o_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="text">
    <string>  Объем упаковки:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserd">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe Print'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #dadada;
color:#000000;
</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2d">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3d">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4d">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5d">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6d">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: #FFE1B5;
color:#000000;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4d">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;</string>
   </property>
   <property name="text">
    <string>Впишите описание вкуса</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
